src/stories/Foundation/StatusTag/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { ErrorSeverity } from '../../../shared/types';
+import styled from 'styled-components';
+import rem from '../../../shared/pxToRem';
+
+const getColorForSeverity: {
+  [key in ErrorSeverity]: { border: string; background: string; color: string };
+} = {
+  warning: { border: 'sunYellow', background: 'cheesecakeYellow', color: 'autumnYellow' },
+  error: { border: 'petitePink', background: 'whisperCoral', color: 'redWine' },
+  success: { border: 'springGreen', background: 'whisperGreen', color: 'oreganoGreen' },
+  neutral: { border: 'grey76', background: 'grey91', color: 'grey35' },
+  info: { border: 'plumbagoBlue', background: 'whisperBlue', color: 'royalBlue' },
+};
+
+const Wrapper = styled.div<{ severity: ErrorSeverity }>`
```

Comment:
Should maybe be a `p` or `span` so it stays inline? Seems a bit more semantic.

Reply:
Agree!